src/components/Common/NotePreview.tsx

```
@@ -0,0 +1,109 @@
+    container?.addEventListener("mouseenter", handleMentionHover, true);
+    container?.addEventListener("mouseleave", handleMentionHover, true);
+
+    return () => {
+      container?.removeEventListener("mouseenter", handleMentionHover, true);
+      container?.removeEventListener("mouseleave", handleMentionHover, true);
+    };
+  }, []);
+
+  return (
+    <div ref={containerRef} className="relative prose text-sm prose-p:m-0">
+      <div dangerouslySetInnerHTML={{ __html: processText(initialNote) }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)